lib/pages/LoginPage.dart

```diff
@@ -29,24 +29,29 @@ class _LoginPageState extends State<LoginPage> {
   @override
   Widget build(BuildContext context) {
     return new Scaffold(
+        backgroundColor: Color.fromARGB(255, 255, 255, 255),
         // resizeToAvoidBottomPadding: false,
         body: Column(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: <Widget>[
           Container(
             child: Stack(
               children: <Widget>[
-                Container(
+                Center(child: Container(
                   padding: EdgeInsets.fromLTRB(15.0, 110.0, 0.0, 0.0),
                   child: Text(
-                    'EVENTO',
+                    'BANDIST',
                     style:
-                        TextStyle(fontSize: 80.0, fontWeight: FontWeight.bold),
+                        TextStyle(fontSize: 70.0, fontWeight: FontWeight.bold),
                   ),
-                ),
+                ),),
               ],
             ),
           ),
+          SizedBox(height: 20.0),
+          Center(
+            child: Image(image: AssetImage('assets/undraw_Compose_music_re_wpiw.png')),
+          ),
```

Comment:
I am experiencing this issue here

<img width="199" alt="image" src="https://user-images.githubusercontent.com/79397892/205433377-46c55872-0830-4459-a0e7-a80381aaf79d.png">

First, you should add path to the right file, which I am assuming is loginImage.png. Also try looking into adding property fit: BoxFit.cover, as well as wrapping into Expanded() widget, otherwise we will have overflow.